nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/content/recipe/HelmHostedFacetImpl.java

```diff
@@ -0,0 +1,71 @@
+package org.sonatype.repository.helm.internal.content.recipe;
+
+import java.io.IOException;
+
+import javax.annotation.Nullable;
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import org.sonatype.nexus.repository.FacetSupport;
+import org.sonatype.nexus.repository.config.Configuration;
+import org.sonatype.nexus.repository.view.Content;
+import org.sonatype.nexus.repository.view.Payload;
+import org.sonatype.repository.helm.internal.AssetKind;
+import org.sonatype.repository.helm.internal.content.HelmContentFacet;
+import org.sonatype.repository.helm.internal.hosted.HelmHostedFacet;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static org.sonatype.repository.helm.internal.AssetKind.HELM_PACKAGE;
+import static org.sonatype.repository.helm.internal.AssetKind.HELM_PROVENANCE;
+
+/**
+ * {@link HelmHostedFacet implementation}
+ *
+ * @since 1.0.10
+ */
+@Named
+public class HelmHostedFacetImpl
+    extends FacetSupport
+    implements HelmHostedFacet
+{
+  private HelmContentFacet helmContentFacet;
+
+  @Inject
+  public HelmHostedFacetImpl()
+  {
+  }
+
+  @Override
+  protected void doInit(final Configuration configuration) throws Exception {
+    super.doInit(configuration);
+    //getRepository().facet(StorageFacet.class).registerWritePolicySelector(new HelmHostedWritePolicySelector());
```

Comment:
Do we need this commented code ?

Reply:
Fixed efa26c88b42be3a59e47c323b99404f3d5fe08af